cfme/tests/containers/test_one_image_analysis.py

```diff
@@ -0,0 +1,63 @@
+# -*- coding: utf-8 -*-
+import pytest
+from cfme.fixtures import pytest_selenium as sel
+from utils import testgen
+from utils.version import current_version
+from cfme.web_ui import flash, toolbar as tb, tabstrip as tabs
+from utils.wait import wait_for, TimedOutError
+from cfme.configure import tasks
+
+# Polarion test case CMP-10064
+pytestmark = [
+    pytest.mark.uncollectif(
+        lambda: current_version() < "5.6"),
+    pytest.mark.usefixtures('setup_provider'),
+    pytest.mark.tier(1)]
+pytest_generate_tests = testgen.generate(
+    testgen.container_providers, scope="function")
+
+
+def test_image_analysis_finished():
+    sel.force_navigate('containers_images')
+    checkbox = ".//input[@class='list-grid-checkbox']"
```

Comment:
Can you move this to model? It is not very good having locators in tests, unless they are testing some specific UI glitch or similar thing.